src/components/ProductCard.tsx

```diff
@@ -10,13 +13,67 @@ type Props = {
 export const ProductCard: FC<Props> = ({ product }) => {
   return (
     <div className={productStyles.card}>
-      <Image
+      <Image 
+        className={productStyles.card__img}
         src={`/${product.image}`}
         alt={product.name}
         width={208}
         height={196}
```

Comment:
it's not nessesarry to write width and height in component. You can do this in css

Reply:
Its required when using absolute path to image